api/v1beta2/ibmpowervscluster_webhook.go

```diff
@@ -97,7 +98,42 @@ func (r *IBMPowerVSCluster) validateIBMPowerVSClusterNetwork() *field.Error {
 	return nil
 }
 
-func (r *IBMPowerVSCluster) validateIBMPowerVSClusterCreateInfraPrereq() *field.Error {
+func (r *IBMPowerVSCluster) validateIBMPowerVSClusterLoadBalancerPort() (allErrs field.ErrorList) {
+	for i, loadbalancer := range r.Spec.LoadBalancers {
+		for j, listerner := range loadbalancer.AdditionalListeners {
+			if listerner.Port == int64(DefaultAPIServerPort) {
+				allErrs = append(allErrs, field.Invalid(field.NewPath("spec", fmt.Sprintf("loadbalancers[%d]", i), fmt.Sprintf("additionalListeners[%d]", j), "port"), listerner.Port, fmt.Sprintf("value of port should not be equal to APIServerPort (%d)", DefaultAPIServerPort)))
```

Comment:
Lets put some more use full message to user like, The 6443 port will be by default configured to loadbalnacer and its not allowed to configure again for something of that type, So user should not think that not able to attach 6443 port to lb.